data_steward/tools/import_rdr_omop.sh

```diff
@@ -83,19 +84,16 @@ fi
 for file in $cdm_files; do
   filename=$(basename ${file})
   table_name=${filename%.*}
+  schema_file="$(find "${FIELDS_DIR}" -type f -iname "${table_name}".json)"
```

Comment:
Looks like you figured it out! I would just suggest that it check that one (and only one) schema is located or otherwise fail.

```suggestion
  # Locate the schema file (ensuring one and only one is found)
  schema_file=$(find "${FIELDS_DIR}" -type f -iname "${table_name}.json")
  schema_count=$(echo "$schema_file" | wc -l);
  if [[ 1 -ne schema_count ]]; then
    echo "Import failed: ${schema_count} schemas were found for table ${table_name}."
    exit 1
  fi
```